index.html

```diff
@@ -14,20 +14,172 @@
+            <button>Ruby on Rails</button>
+            <button> Javascript</button>
+            <div>
+              <button class="go-to dark-bg">See this project
+                <i class="fas fa-arrow-right"></i>
+              </button>
+            </div>
+          </div>
+        </div>
+        <div class="cards">
+          <div class="card3 fix-card"></div>
+          <div class="cards-detail dark-bg">
+            <p class="poppins">name goes here</p>
+            <button>HTML/CSS</button>
+            <button>Ruby on Rails</button>
+            <button> Javascript</button>
+            <div>
+              <button class="go-to dark-bg">See this project
+                <i class="fas fa-arrow-right"></i>
+              </button>
+            </div>
+          </div>
+        </div>
+        <div class="cards">
+          <div class="card4 fix-card"></div>
+          <div class="cards-detail dark-bg">
+            <p class="poppins">name goes here</p>
+            <button>HTML/CSS</button>
+            <button>Ruby on Rails</button>
+            <button> Javascript</button>
+            <div>
+              <button class="go-to dark-bg">See this project
+                <i class="fas fa-arrow-right"></i>
+              </button>
+            </div>
+          </div>
+        </div>
+        <div class="cards">
+          <div class="card5 fix-card"></div>
+          <div class="cards-detail dark-bg">
+            <p class="poppins">name goes here</p>
+            <button>HTML/CSS</button>
+            <button>Ruby on Rails</button>
+            <button> Javascript</button>
+            <div>
+              <button class="go-to dark-bg">See this project
+                <i class="fas fa-arrow-right"></i>
+              </button>
+            </div>
+          </div>
+        </div>
+        <div class="cards">
+          <div class="card6 fix-card"></div>
+          <div class="cards-detail dark-bg">
+            <p class="poppins">name goes here</p>
+            <button>HTML/CSS</button>
+            <button>Ruby on Rails</button>
+            <button> Javascript</button>
+            <div>
+              <button class="go-to dark-bg">See this project
+                <i class="fas fa-arrow-right"></i>
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section> 
+
+  <section class="about-section">
+    <div class="primary-color about-head position typography left16">
+      <h2>About</h2>
+    </div>
+    <div class="about left16 position poppins">
+      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+        labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod</p>
+    </div>
+
+    <div>
+      <button class="cta-2 dark-bg position left16 width288 poppins">Get my CV
+        <i class="fas fa-download"></i>
+      </button>
```

Comment:
- [ ] Please add an `a` tag to this button and give it a hover effect so that a scrolling user can easily tell that it is a clickable button.